tests/test_precision.py

```diff
@@ -78,7 +78,7 @@ def predict_and_measure_memory(precision) -> int:
 def test_train_precision_memory(precision: Precision):
     memory_fp32 = fit_and_measure_memory(Precision.FP32)
     memory_half = fit_and_measure_memory(precision)
-    assert memory_half < 0.87 * memory_fp32
+    assert memory_half < 0.89 * memory_fp32
```

Comment:
Torch 2.7 passes when I set a slightly higher value; this seems empirical by nature so I don't think this should be too big of an issue?

Reply:
yeah agreed, ok with this one. Seems reasonable that there are variations in memory with torch/cuda bump